assets/css/hazard-card.css

```diff
@@ -0,0 +1,63 @@
+.sb-cards-btn--back {
+  display: flex;
+  align-items: center;
+  color: var(--neutral-neutral-900);
+  border-radius: 100px;
+  border: 1px solid var(--neutral-neutral-300);
```

Comment:
double-check variable

Reply:
Copied these names from Figma (Dev tab). I will check all the variables and change accordingly